about.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lato"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Microverse Concert</title>
+  </head>
+  <body>
+    <section class="navigation">
+      <nav class="top-bar">
+        <i class="fa fa-facebook"></i>
+        <i class="fa fa-instagram"></i>
+        <i class="fa fa-youtube"></i>
+        <a href="#">Login</a>
+      </nav>
+      <nav class="flex-nav">
+        <div class="menu-open">
+          <img src="assets/img/ic_menu.svg" alt="" />
+        </div>
+        <div class="menu">
+          <a href="index.html" class="logo"
+            ><img src="assets/img/logo.png" alt=""
+          /></a>
+          <a href="about.html">About</a>
+          <a href="#media">Media</a>
+          <a href="#lineup">Line up</a>
+          <a href="#buytickets">Buy tickets</a>
+          <div class="menu-close">
+            <img src="assets/img/Union.svg" alt="" />
+          </div>
+        </div>
+      </nav>
+    </section>
+    <header class="about about-bg">
+      <p class="hello about">"Hello Micronauts!"</p>
+      <h1 class="title about">Microverse Concert Music for Developers 2021</h1>
+      <div class="intro about">
+        <p>
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Officiis
+          nulla animi praesentium cumque hic! Est quis dolorum quidem itaque,
+          voluptatum impedit expedita. Minus itaque nisi magnam perferendis
+          quas? <br />
+          Incidunt provident rem accusamus adipisci, quisquam perspiciatis
+          temporibus! Rerum sapiente dolores possimus unde odio nostrum minima,
+          inventore reprehenderit commodi debitis perspiciatis saepe aliquam
+          impedit molestiae culpa dignissimos reiciendis in soluta! <br />
+          Apestius dolor sit, amet consectetur adipisicing elit. Recusandae,
+          sunt.
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Accus antium
+          ipsam iste itaque nihil architecto, volu tatem sed consectetur
+          adipisicing elit facilis porro. Inventore necessitatibus suscipit
+          distinctio quidem dolorem placeat! Ad autem nisi vel repellendus.
+          Iste!
+        </p>
+      </div>
```

Comment:
- [ ] Since this section doesn't show up on the about page, there's no point in adding it to the HTML file. Kindly remove it.